bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingImpl.java

```diff
@@ -43,6 +48,8 @@
 
     private Configuration configuration = new Configuration();
 
+    private Map<String, String> properties = new HashMap<>();
```

Comment:
Yes exactly. The Configuration contains more logic, like "as" for transforming into a bean and also a method "isProperlyConfigured" (which being honest is not used anywhere and return always true). But I could not imagine any additional logic for the properties.